prime-router/metadata/HL7/catchall/hl7/segments/IN1/IN1Extension.yml

```diff
@@ -0,0 +1,701 @@
+          type: STRING
+          valueOf: IN1.26
+          expressionType: HL7Spec
+    - expressionType: nested
+      specs: IN1.27
+      condition: $in127 NOT_NULL
+      generateList: true
+      vars:
+        in127: STRING_ALL, IN1.27
+      expressionsMap:
+        url:
+          type: STRING
+          value: IN1.27
+        valueCodeableConcept:
+          valueOf: datatypes/CWE/CodeableConcept
+          expressionType: resource
+    - expressionType: nested
+      vars:
+        in128: STRING_ALL, IN1.28
+      condition: $in128 NOT_NULL
+      specs: IN1.28
+      generateList: true
+      expressionsMap:
+        url:
+          type: STRING
+          value: IN1.28
+        valueString:
+          type: STRING
+          valueOf: IN1.28
+          expressionType: HL7Spec
+    - expressionType: nested
+      vars:
+        in129: IN1.29
+      condition: $in129 NOT_NULL
+      generateList: true
+      expressionsMap:
+        url:
+          type: STRING
+          value: IN1.29
+        valueDateTime:
+          expressionType: nested
+          expressions:
+            - type: STRING
+              valueOf: "GeneralUtils.dateTimeWithZoneId(in129,ZONEID)"
+              expressionType: JEXL
+        _valueDateTime:
+          expressionType: nested
+          expressionsMap:
+            extension:
+              generateList: true
+              expressionType: nested
+              expressionsMap:
+                url:
+                  type: SYSTEM_URL
+                  value: hl7v2-date-time
+                valueString:
+                  type: STRING
+                  valueOf: IN1.29
+                  expressionType: HL7Spec
+    - expressionType: nested
+      vars:
+        in130: STRING_ALL, IN1.30
+      condition: $in130 NOT_NULL
+      specs: IN1.30 *
+      generateList: true
+      expressionsMap:
+        url:
+          type: STRING
+          value: IN1.30
+        valueReference:
+          expressionType: reference
+          valueOf: datatypes/XCN/Practitioner
+    - expressionType: nested
+      specs: IN1.31
+      condition: $in131 NOT_NULL
+      generateList: true
+      vars:
+        in131: STRING_ALL, IN1.31
+      expressionsMap:
+        url:
+          type: STRING
+          value: IN1.31
+        valueCodeableConcept:
+          valueOf: datatypes/CWE/CodeableConcept
+          expressionType: resource
+    - expressionType: nested
+      specs: IN1.32
+      condition: $in132 NOT_NULL
+      generateList: true
+      vars:
+        in132: STRING_ALL, IN1.32
+      expressionsMap:
+        url:
+          type: STRING
+          value: IN1.32
+        valueCodeableConcept:
+          valueOf: datatypes/CWE/CodeableConcept
+          expressionType: resource
+    - expressionType: nested
+      vars:
+        in133: IN1.33
+      condition: $in133 NOT_NULL
+      generateList: true
+      expressionsMap:
+        url:
+          type: STRING
+          value: IN1.33
+        valueString:
```

Comment:
Should we consider mapping this and IN1.34 to `valueDecimal` instead? 
Edit: On second thought all these extension fields fall into "whose line is it anyway" territory. So there may be more accurate ways to represent it but unless anyone is attempting send these fields it's going to make no difference.

Reply:
That second thought was my thought process on how to handle `NM`, `ID`, etc. since our experience has been that basically anything can be put in these fields regardless of how the specification says they should be used, and storing a simple string representation is the simplest way to carry that all the way through the pipeline.

Reply:
That's a really good point! Getting more specific with the value format might be accurate but it forces us to be less flexible on fields that we can't really predict how they will be used.